config.json

```diff
@@ -703,7 +703,13 @@
         "name": "Perfect Numbers",
         "uuid": "c6691fd2-e10d-47df-acbf-3adeac5a2f89",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "numbers",
+          "arrays",
+          "conditionals",
+          "errors",
+          "for-loops"
+        ],
```

Comment:
Here is my solution : https://exercism.org/tracks/javascript/exercises/perfect-numbers/solutions/TomPradat